src/System.Private.CoreLib/shared/System/SpanHelpers.Byte.cs

```diff
@@ -1577,48 +1577,16 @@ private static int LocateLastFoundByte(Vector<byte> match)
         private static int LocateFirstFoundByte(ulong match)
         {
             // TODO: Arm variants
-            if (Bmi1.X64.IsSupported)
-            {
-                return (int)(Bmi1.X64.TrailingZeroCount(match) >> 3);
-            }
-            else
-            {
-                // Flag least significant power of two bit
-                var powerOfTwoFlag = match ^ (match - 1);
```

Comment:
Is the BitOpts operation as fast as this?

Reply:
Benched - no, was 1.2x slower. Reverted.